packages/kit/src/runtime/app/server/event.js

```diff
@@ -0,0 +1,46 @@
+/** @import { RequestEvent } from '@sveltejs/kit' */
+
+/** @type {RequestEvent | null} */
+let request_event = null;
+
+/** @type {import('node:async_hooks').AsyncLocalStorage<RequestEvent>} */
+let als;
+
+try {
+	const hooks = await import('node:async_hooks');
+	als = new hooks.AsyncLocalStorage();
+} catch {
+	// can't use AsyncLocalStorage, but can still call getRequestEvent synchronously.
```

Comment:
what do users see on StackBlitz? Will it be the `Can only read the current request event when the event is being processed` message? That might be confusing if so and perhaps it's better to throw a different error here

Reply:
upgraded the message to include a mention of ALS, if it's absent